pages/resources/opportunities.jsx

```diff
@@ -124,13 +124,17 @@ export default function Opportunities({ opps, ...props }) {
               ) : (
                 <>
                   {!!oppPosts &&
-                    oppPosts.map(({ postingName, slug }) => (
+                    oppPosts.map(({ postingName, orgImages, orgName, city, postedDate, slug }) => (
                       <div key={postingName}>
-                        <Link href={`/resources/opportunities/${slug}`}>
-                          <a style={{ color: baseTheme.colors.navy }}>
-                            {postingName}
-                          </a>
-                        </Link>
+												<OpportunityCard
+													name={postingName}
+													link={`/resources/opportunities/${slug}`}
+													organization={orgName}
+													location={city}
+													category="placeholder"
+													postingDate={postedDate}
+													logo={orgImages}
+												/>
```

Comment:
Wrap this in 
```
<Link href=`/resources/opportunities/${slug}`>
  <a>
    <OpportunityCard ... /> 
  </a> 
</Link>
``` 
to link the card instead of the opportunity title

Reply:
See my comment on how this would interact with assistive technologies: I might end up doing this anyways, but it's something I wanted to bring up for consideration just in case it's something we need to address

Reply:
Sure, we can probably discuss just linking the blog/opportunity title. This is just if you're having trouble with linking the whole card 
>  I need to make the cards clickable, to access each opportunity. As of right now just the opportunity names are clickable links, because I threw this together very quickly.